assets/javascript/logic.js

```diff
@@ -245,6 +241,20 @@ function updateRecipeBox() {
   }
 }
 
+//a function that takes in a time string from the recipe object and retuns a formatted time string for display
+//the formatted time string is <hours> 'hours' + <minutes> 'minutes'
+function formatTime(time) {
+  //turn the time string into an integer and compute the hours and minutes
+  var timeInMinutes = parseint(time);
+  //only compute the hours and minutes if time is nonzero
+  if (timeInMinutes !== 0) {
+    var hours = Math.floor(timeInMinutes / 60);
+    var minutes = Math.round(timeInMinutes % 60);
+    return hours + " hours " + minutes + " minutes";
+  }
+  return "no time data exists for this recipe";
```

Comment:
I would vote for returning "" as anything else would be repeated a lot. Meaning if there is no time value we just don't mention it.

Reply:
hmm. Interesting thought. I'm indifferent.